### PR TITLE
FOCUS #38: Create a Billed Cost metric

### PR DESCRIPTION
**Pull request by chandra-cloudtrakr**

### CODE REVIEW

File: specification/metrics/billedcost.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Billed Cost
+
+Billed Cost represents a charge inclusive of negotiated discounts that a consumer would be charged for each billing period. Billed Cost does not amortize up-front charges (one-time or recurring). Billed Cost is often used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
```
Comment (flanakin) [inline specification/metrics/billedcost.md:3]:
```suggestion
The billed cost represents a charge inclusive of negotiated discounts that a consumer would be charged for each billing period. Billed cost does not amortize up-front charges (one-time or recurring). Billed cost is often used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
```

Comment (flanakin) [inline specification/metrics/billedcost.md:3]: Don't organizations use amortized cost for cost allocation?

File: specification/metrics/billedcost.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+See [Supporting content](https://docs.google.com/document/d/1dwrGmkzpP-NfjZPFzv8HiSo_LBgg2serBfSKFZx4Lnk/edit#heading=h.5s5fte4oc77p)
```
Comment (flanakin) [inline specification/metrics/billedcost.md:5]: Shouldn't this be a local file link?

File: specification/metrics/billedcost.md (hunk 1 above)
Comment (udam-f2) [reply specification/metrics/billedcost.md:3]: Only if a central team is paying the up-front purchase cost. We used to, and charge amortized cost + cost of funds but that's not always the case.

Comment (SanjnaSrivatsaVMware) [reply specification/metrics/billedcost.md]: Isn't upfront 1 word?

Comment (flanakin) [reply specification/metrics/billedcost.md]: Yes, 1 word.

File: specification/metrics/billedcost.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Billed Cost
+
+The billed cost represents a charge inclusive of negotiated discounts that a consumer would be charged for each billing period. Billed cost does not amortize upfront charges (one-time or recurring). Billed cost is often used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
+
+See [Supporting Content](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/23-billed-cost-metric/supporting_content/metrics/billedcost.md) for examples of billing data from the cloud providers.
```
Comment (udam-f2) [inline specification/metrics/billedcost.md:5]:
```suggestion
The BilledCost column MUST be present in the billing data. This column MUST be a numeric value of type Decimal and MUST NOT contain null values. The aggregated BilledCost for a billing period MUST match the charge received on the invoice for the same billing period. 
```

File: supporting_content/metrics/billedcost.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+#### Example usage scenarios
+
+See [Cost Metrics Examples Spreadsheet](https://docs.google.com/spreadsheets/d/1bhRELDgf3LTSfQJRrCyovTt65g4ElimYHq6fmKOz83E) for examples of billing data for various scenarios.
+
+Current values observed in billing data for various scenarios:
+
+| **Provider** | **Scenario** | **Pattern** |
+|---|---|---|
+| AWS | No up-front RI | BilledCost will represent the RI Fee charged per day for the RI (for usage). Instance that had its usage cost discounted will show up as \$0 for BilledCost. |
+
```
Comment (udam-f2) [inline supporting_content/metrics/billedcost.md:24]:
```suggestion
```

File: supporting_content/metrics/billedcost.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+| **BilledCost** | **BilledCurrency** | **PricedCost** | **PricedCurrency** |
+|---|---|---|---|
+| 123.45 | USD | 123.45 | USD |
+| 246.80 | USD | 246.80 | USD |
+| 1234567890 | JPY | 89.08 | USD |
+| 369.15 | CNY | 369.15 | CNY |
+
+*Do we just need a THIRD option - CostInBase, BaseCurrency - which is like UTC - it could be USD? OR are we assuming that PricedCurrency is always USD?*
+
+- List price is 5\$ comes in hourly. Providers may decide to modify that cost for various reasons (negotiated discounts, commitment discounts etc.).
```
Comment (udam-f2) [inline supporting_content/metrics/billedcost.md:61]:
```suggestion
```

**Review by udam-f2 — CHANGES_REQUESTED** [review]

**Review by udam-f2 — APPROVED** [review]
LGTM

**Pull request merged**